test/test_manifest.json

```diff
@@ -150,7 +150,7 @@
     },
     {  "id": "intelisa-eq",
        "file": "pdfs/intelisa.pdf",
-       "md5": "24643ebe348a568cfe6a532055c71493",
+       "md5": "659e9dd1173cbc822ed7540547f8a947",
```

Comment:
Since it appears that this PDF file changes every now and then, I'd suggest that we link to a *particular* version on the Internet Archive instead, to prevent new users from getting unnecessary warnings about md5 mismatches when running tests.

Unfortunately the old version doesn't seem to be available at the Internet Archive, but this one looks close enough: http://web.archive.org/web/20130302223208/http://www.intel.com/content/dam/www/public/us/en/documents/manuals/64-ia-32-architectures-software-developer-vol-1-manual.pdf.